resources/aws/instance.go

```diff
@@ -150,6 +154,7 @@ func (i *Instance) CreateOrFail() error {
 			SubnetId: aws.String(i.SubnetID),
 		})
 		if err != nil {
+			i.Logger.Log("error", fmt.Sprintf("creating instance failed, retrying: %v", errgo.Details(err)))
```

Comment:
FYI, this is not how this should be used. See https://godoc.org/github.com/cenkalti/backoff#RetryNotify. There is a notifier that should be configured. This notifier can deal with logging and metrics and the like.